init.fish

```diff
@@ -33,9 +33,9 @@ function fish_user_key_bindings
     and read -l theme < $OMF_CONFIG/theme
     or set -l theme default
   # Prepare packages key bindings paths
-  set -l key_bindings $OMF_CONFIG/key_binding?.fish \
+  set -l key_bindings $OMF_CONFIG/key_binding*.fish \
```

Comment:
should we move the `*` on this or the `off.theme.set` at the bottom?

Reply:
I'm looking into that.

Also, it's `key_bindings.fish`, with a trailing `s`, at least for the plugins and themes that have it:

* [plugin-bang-bang](/oh-my-fish/plugin-bang-bang)
* [plugin-expand](/oh-my-fish/plugin-expand)
* [plugin-sudope](/oh-my-fish/plugin-sudope)
* [theme-budspencer](/oh-my-fish/theme-budspencer)
* [theme-dangerous](/oh-my-fish/theme-dangerous)

Reply:
Having `$OMF_CONFIG*/key_bindings.fish` leads to problem in case of `~/.config/omf_old`, for example.

Reply:
Maybe there's no way to make it part of `$key_bindings` like the others. What if we solve it with the following?

```fish
test -e $OMF_CONFIG/key_bindings.fish; and source $OMF_CONFIG/key_bindings.fish
```